src/hometask6/the_decorator_new.py

```diff
@@ -0,0 +1,66 @@
+# Создайте декоратор, который вызывает задекорированную
+# функцию пока она не выполнится без исключений (но не
+# более n раз - параметр декоратора).
+# Если превышено количество попыток,
+# должно быть возбуждено исключение типа TooManyErrors
+
+from functools import wraps
+import the_functions_new
+
+
+class TooManyErrors(Exception):
+    def __init__(self, function_name):
+        self.function_name = function_name
+
+    def __str__(self):
+        return 'TooManyErrors at {}'.format(self.function_name)
+
+
+def error_decorator(critical_error_count):
```

Comment:
I expected it would  for one function call try to call it not more than critical_err_count times.

Reply:
Fixed